ls1intum/Orion
# `Enhancement`: Delete trailing slashes

### Motivation, Context and Description
Fixes #75. Originally I settled with a highlighted note in Orion's documentation, but as pointed out this is not immediately obvious to new users. Since I failed to find a generic approach to parse the url and generate a normalized form using the java library, this PR now just bluntly removes any trailing slash, solving the issue at hand, but leaving other, similar issues with urls written in other formats as is.

Also removes a document listener setting a variable that was never read, presumably a leftover from a previous refactoring

### Steps for Testing


1.  Install the release as described in [the readme](https://github.com/ls1intum/Orion/blob/master/README.md#testing-of-pull-requests).
2.  Configure a URL with a trailing slash as Artemis Base URL.
3.  Bear in mind only adding the slash will not change the stored value and therefore not trigger a reload.
4.  All pages should still load normally.
